<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiGestionPhoto</class>
 <widget class="QWidget" name="GuiGestionPhoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>88</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Saisie de l'url de la photo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblUrl">
       <property name="text">
        <string>Adresse URL :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSaisieUrl"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnValider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../../SeboApp.qrc">
         <normaloff>:/SeboApp/Resources/valider.png</normaloff>:/SeboApp/Resources/valider.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../../SeboApp.qrc">
         <normaloff>:/SeboApp/Resources/Annuler.png</normaloff>:/SeboApp/Resources/Annuler.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../SeboApp.qrc"/>
 </resources>
 <connections/>
</ui>
